packages/typewiz-core/src/configuration-parser.ts

```diff
@@ -0,0 +1,65 @@
+import * as Ajv from 'ajv';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+import * as util from 'util';
+import { IApplyTypesOptions } from './apply-types';
+import { ICompilerOptions } from './compiler-helper';
+import { IInstrumentOptions } from './instrument';
+const readFileAsync = util.promisify(fs.readFile);
+
+export class ConfigurationParser {
+    private typewizConfig: any;
+    private configurationPath: string | undefined;
+
+    public findConfigFile(cwd: string): string {
+        return ts.findConfigFile(cwd, ts.sys.fileExists, 'typewiz.json');
+    }
+
+    public async parse(configurationPath: string): Promise<void> {
+        const typewizConfigSchema = JSON.parse(
+            await readFileAsync(path.join(__dirname, 'typewiz.json'), { encoding: 'utf8' }),
```

Comment:
Since the schema is not going to change in runtime, why not read it the the beginning of the file using `require('./typewiz.json')` ? This will also eliminate the need for `JSON.parse`

Reply:
Done

Reply:
Thanks 👍